src/screens/profile/styles.ts

```diff
@@ -3,7 +3,7 @@ import {Dimensions, StyleSheet} from 'react-native';
 export default StyleSheet.create({
   container: {
     flex: 1,
-    justifyContent: 'flex-end',
+    justifyContent: 'center',
```

Comment:
didn't this change the position of the LogOut button on the screen?

Reply:
i checked it at first doing it and now, and i don't see changes, it stays at top right